cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -163,13 +169,14 @@ def resource_usage(vm_ownership, db, provider, ssh_client_modscope):
     # Chargeback reporting is not done on real-time values.So, we are capturing C&U data
     # and forcing hourly rollups by running these commands through the Rails console.
 
-    logger.info('CAPTURING PERF DATA FOR VM {} running on {}'.format(vm_name, provider.name))
     ssh_client_modscope.run_rails_command(
         "\"vm = Vm.where(:ems_id => {}).where(:name => {})[0];\
         vm.perf_capture('realtime',1.hour.ago.utc, Time.now.utc);\
         vm.perf_rollup_range('realtime',1.hour.ago.utc, Time.now.utc)\"".
         format(provider_id, repr(vm_name)))
-    wait_for(count_records_rollups_table, [db, provider], timeout=60, fail_condition=0,
+    logger.info('WAITING FOR HOURLY ROLLUP RECORD FOR VM {} running on {}'.format(vm_name,
```

Comment:
A lot of info gets logged to the log/cfme.log file.I've added All- caps, just so that these messages stand out and that makes debugging/troubleshooting a lot easier :).